src/client.ts

```diff
@@ -3424,6 +3424,23 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         return this.getIgnoredUsers().includes(userId);
     }
 
+    /**
+     * Gets the list of currently ignored invites.
+     */
+    public getIgnoredInvites(): IgnoredInvites {
```

Comment:
apologies, should have caught this in the first round of review: this function and the setter will need to be named `unstable_getIgnoredInvites` (and similar for the setter) to make it obvious that the functions can be ripped out. 

Additionally, the return value for both will need to be documented in the jsdoc - see other functions for examples